modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -14,10 +14,11 @@ Anypoint API Manager (API Manager) is a component of Anypoint Platform that enab
 
 API Manager is tightly integrated with the following tools:
 
-* xref:design-center::index.adoc[Anypoint Design Center (Design Center)], for specifying the structure of the API
-* xref:exchange::index.adoc[Anypoint Exchange (Exchange)], for storing and publishing API assets
-* xref:studio::index.adoc[Anypoint Studio (Studio)], for implementing the API
-* xref:runtime-manager::index.adoc[Anypoint Runtime Manager (Runtime Manager)], to deploy, manage, and monitor the API (or proxy)
+* xref:design-center::index.adoc[Design Center]: To specify the structure of the API
+* xref:exchange::index.adoc[Exchange]: To store and publish API assets
+* xref:studio::index.adoc[Studio]: To implement the API
+* xref:runtime-manager::index.adoc[Runtime Manager]: To deploy, manage, and monitor API instances
+* xref:api-governance::index.adoc[API Governance]: To govern aspects of API instances
```

Comment:
```suggestion
* xref:api-governance::index.adoc[API Governance]: To govern API instances
```

Reply:
Instances is one aspect of an API, so the way this is worded now is not correct. Instances don't have aspects. This is a little tough, since aspect is an English word, but we want to use it consistently so as not to confuse the readers.

Reply:
See the definitions [here](https://dev-docs-internal.kqa.msap.io/beta-apig-apim/api-governance/#api-governance-concepts) (and also in the main doc site [Glossary](https://dev-docs-internal.kqa.msap.io/beta-apig-apim/general/glossary)).